libr/include/r_agraph.h

```diff
@@ -32,6 +33,7 @@ typedef struct r_ascii_node_t {
 #define R_AGRAPH_MODE_SUMMARY 4
 #define R_AGRAPH_MODE_COMMENTS 5
 #define R_AGRAPH_MODE_MAX 6
+#define R_AGARPH_MODE_DIFF 7
```

Comment:
shouldn't be `MAX` be the last available value?

Reply:
I have removed this entirely now :)